assembl/static2/package.json

```diff
@@ -146,6 +148,17 @@
   },
   "//": "See https://facebook.github.io/jest/docs/webpack.html for more info",
   "jest": {
+    "coverageReporters": [
```

Comment:
only for front-end!

Reply:
Not sure what you mean, this is jest configuration so it obviously is only about frontend. Like I said this PR is only about frontend and it was decided that backend will be taken care of separately. Gitlab is not compatible with coveralls sadly

Reply:
:+1:

Reply:
we need to push the new values on git!? it's dangerous...

Reply:
Why is that?